general/scans/detector.py

```diff
@@ -29,6 +30,25 @@ def __exit__(self, typ, value, traceback):
         pass
 
 
+class BlockDetector(DetectorManager):
+    """
+    A helper class for using an IBEX block as a detector.
+    """
+    def __init__(self, blockname):
+        self.blockname = blockname
+        self._f = lambda: g.cget(self.blockname)["value"]
```

Comment:
Could this be wrapped in a `try-except` block in case the block doesn't exist? Or will genie python do that for you?